### ISSUE TITLE
Consider marking `RenderHtml` #[must_use]

### ISSUE THREAD

**Issue by TheNeikos** [Altair-Bueno/axum-template#32]
I've ran into this situation:

```
async fn handler(
    engine: impl TemplateEngine,
) -> impl IntoResponse {
    let key = "Template key";
    let data = Person{ /* */ };
    RenderHtml(key, engine, data);
}
```

This renders nothing on the output and also produces no warnings!

Since [`RenderHtml`](https://docs.rs/axum-template/0.19.0/axum_template/struct.RenderHtml.html) is not marked `#[must_use]` the compiler happily ignored it and used the default return value of () as my `IntoResponse` implementation.

Having `#[must_use]` here would at least tell me something is up, and fix the error. (Since `;` is also fairly small and easy to miss)

**Comment by Altair-Bueno** [comment]
Thats unfortunate. It was an oversight from me. It will be fixed on `1.0.0`.

**Comment by Altair-Bueno** [comment]
- Release: https://github.com/Altair-Bueno/axum-template/releases/tag/v1.0.0
- crates.io: https://crates.io/crates/axum-template/1.0.0
- docs.rs: https://docs.rs/axum-template/1.0.0
